edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/RocksDbOptionsProvider.cs

```diff
@@ -44,16 +44,29 @@ public class RocksDbOptionsProvider : IRocksDbOptionsProvider
         // Once write-ahead logs exceed this size, we will start forcing the flush of
         // column families whose memtables are backed by the oldest live WAL file
         const ulong DefaultMaxTotalWalSize = 512 * 1024 * 1024;
+        // Logging to ONE file as a default
+        const ulong DefaultMaxLogFileNum = 1;
```

Comment:
Can we set this to 0?

Reply:
0 would be a default value to RocksDB which is interpreted as no-limits.